docs/usage/configuration-options.md

```diff
@@ -1166,6 +1166,19 @@ If the value starts with `http(s)` then it will only match against URLs which st
 Otherwise, it will be matched by checking if the URL's hostname matches the `matchHost` directly or ends with it.
 When checking the end of the hostname, a single dot is prefixed to the value of `matchHost`, if one is not already present, to ensure it can only match against whole domain segments.
 
+Short hostnames like `domain.com` only work for registries without paths.
```

Comment:
```suggestion
Short hostnames like `domain.com` only work for npm registries without paths.
```

Reply:
I'm not sure either of these are correct

Reply:
The `matchHost` must match the `registryUrl` configured in `.npmrc`, otherwise you'll get authentication issues when the artifacts are updated (yarn / npm/ ... run)